packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -40,59 +44,29 @@ function getBlockDisplayName( blockType, attributes ) {
 	return formatlessDisplayName;
 }
 
-export default function BlockNavigationList( {
-	blocks,
-	selectedBlockClientId,
-	selectBlock,
-	showAppender,
-
-	// Internal use only.
-	showNestedBlocks,
-	parentBlockClientId,
-} ) {
-	const shouldShowAppender = showAppender && !! parentBlockClientId;
+function NavigationList( { blocks, selectBlock, selectedBlockClientId, showAppender, showBlockMovers, showNestedBlocks, parentBlockClientId } ) {
```

Comment:
nit: I liked having each prop on a new line 🙂

Reply:
They're back on a new line in `BlockNavigationList` and `NavigationBlock` has fewer props.